KGTorrent/populate_db.py

```diff
@@ -35,7 +35,7 @@ def check_table_emptiness(table_name, engine):
             return False
         else:
             return True
-
+    """
```

Comment:
Hi Giovanni, here it seems to me that you misused the multi-line string syntax. Multi-line strings in Python are marked by triple quotes `"""`; when they are not used in the context of assignments, they usually serve as multi-line docstrings. On the other hand, they are not meant to be used as multi-line comments, although they are functionally equivalent to them, being ignored by Python interpreters.

PEP8 suggests using dashes `#` for both single-line and multi-line comments and in this project we stick to the PEP8 coding standard. PyCharm is PEP8 compliant too: toggling comments in multi-line selections results in prepending lines with dashes.

---

By the way, in this specific context, I encourage you to delete this function if it is no longer used. Should it turn useful again in the future, we would still be able to restore it from former commits.

Reply:
I'm really sorry Luigi, i forgot to tidy up triple quotes. That was a quick check to see if it is an unused function. As you sudgest i'm going to remove it and i'll be using `#` from now in order to comply with the standards.